auth-gateway/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "auth-gateway",
+  "version": "0.2.9",
```

Comment:
I don't see how this is going to be meaningfully used, so I don't think it matters.

Reply:
I don't want to have to edit the Hail version in multiple places when we bump it.  That's a major anti-pattern we learned during the 0.1 => devel => 0.2 version changes (and still hasn't been resolved).  I can basically guarantee this will be out of date within a couple week's time.